scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -134,6 +139,11 @@
     /** displayed when critical log events occur */
     private ToolStripButton errorButton = null;
 
+    /** displays the scheduler status */
+    private HLayout schedulerStatusLabelLayout = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "schedulerStatusLabelLayout" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)